### PR TITLE
style: add more whitespace around footer

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**

### CODE REVIEW

File: resources/js/components/search/ArtworksMasonry.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
             <button
                 v-if="page === 1"
                 @click="page++"
-                class="uppercase font-medium py-2.5 px-6 m-2 lg:m-4 leading-[1.125] border-black border-2"
+                class="uppercase font-medium py-2.5 px-6 mt-10 leading-[1.125] border-black border-2"
```
Comment (eronisko) [inline resources/js/components/search/ArtworksMasonry.vue:16]: We're losing horizontal margins here

Comment (FrantisekMichalSebestyen) [reply resources/js/components/search/ArtworksMasonry.vue:16]: are they used ?

Comment (eronisko) [reply resources/js/components/search/ArtworksMasonry.vue:16]: I don't know, are they? :) I haven't got the code checked out. If they're not being used, then fine, just please let me know

Comment (FrantisekMichalSebestyen) [reply resources/js/components/search/ArtworksMasonry.vue:16]: Ah yes :D didn't mean to be rude, sorry. Was just curious if you know something I don't.

**Review by eronisko — APPROVED** [review]

**Pull request merged**